products/redhat.md

```diff
@@ -49,3 +80,11 @@ releases:
 Red Hat Enterprise Linux versions 5, 6, and 7 each deliver ten years of support in Full Support, Maintenance Support 1 and Maintenance Support 2 Phases followed by an Extended Life Phase. In addition, for Red Hat Enterprise Linux 5 and 6, customers may purchase annual Add-on subscriptions called Extended Life-cycle Support (ELS) to extend limited subscription services beyond the Maintenance Support 2 Phase.
 
 With the introduction of Red Hat Enterprise Linux version 8, Red Hat is simplifying the RHEL product phases from four to three: Full Support, Maintenance Support, and Extended Life Phase.
+
+* **Full Support Phase:** During the Full Support Phase, qualified Critical and Important Security errata advisories (RHSAs) and Urgent and Selected High Priority Bug Fix errata advisories (RHBAs) may be released as they become available. If available, new or improved hardware enablement and select enhanced software functionality may be provided at Red Hat’s discretion.
+
+* **Maintenance Support Phase:** During the Maintenance Support Phases, qualified RHSAs and RHBAs may be released as they become available. Other errata advisories may be delivered as appropriate. New functionality and new hardware enablement is at the discretion of Red Hat, and varies by operating system release.
+    
+* **Extended Life Phase (ELS):** During the Extended Life Phase, a Red Hat Enterprise Linux subscription provides continued access to previously released content on the Red Hat Customer Portal, as well as other content such as documentation and the Red Hat knowledgebase. By default no bug fixes, security fixes, hardware enablement, or root-cause analysis is available during this phase. This is also not available for the architectures System z (Structure A), ARM, and POWER9 on RHEL 7.
```

Comment:
Let's move the last line from here, since that's only about RHEL7. Maybe reword it as a quirk for RHEL 7 instead:

> RHEL 7 (ELS) does not cover a,b,c architectures.

Reply:
> If ELS does not provide any fixes, this is just to track documentation availability? Would tracking this provide a false sense of relief to users on the main table?

ELS does provide certain security fixes as mentioned in: https://www.redhat.com/en/resources/els-datasheet

I added a small description of this in https://www.redhat.com/en/resources/els-datasheet

Reply:
> Let's move the last line from here, since that's only about RHEL7. Maybe reword it as a quirk for RHEL 7 instead:
> 
> > RHEL 7 (ELS) does not cover a,b,c architectures.

See latest version for this,